test/spec/modules/outbrainBidAdapter_spec.js

```diff
@@ -522,6 +524,25 @@ describe('Outbrain Adapter', function () {
         ]);
       });
 
+      it('should pass OB user token', function () {
+        getDataFromLocalStorageStub = sinon.stub(storage, 'getDataFromLocalStorage');
+        getDataFromLocalStorageStub.returns('12345');
+
+        let bidRequest = {
+          bidId: 'bidId',
+          params: {},
+          ...commonBidRequest,
+        };
+
+        let res = spec.buildRequests([bidRequest], commonBidderRequest);
+        const resData = JSON.parse(res.data)
+        expect(resData.user.ext.obusertoken).to.equal('12345')
+        expect(getDataFromLocalStorageStub.calledOnce).to.be.true;
+        sinon.assert.calledWith(getDataFromLocalStorageStub, 'OB-USER-TOKEN');
+
+        getDataFromLocalStorageStub.restore();
+      });
```

Comment:
hey, thanks for the suggestion! I don't want to stub return of the method before each test because I don't want to fix other tests (the response should stay the same if we don't read anything from local storage). So I hope this is ok
[9fd5d42](https://github.com/prebid/Prebid.js/pull/10382/commits/9fd5d42ef728491a96f5e5f1d3354ecc18e42530)